### DISCUSSION TITLE
Certificate not working for www redirect

### DISCUSSION THREAD

**Discussion by nmeijers31** [community/community#57212] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I am trying to redirect my www address to the naked domain but have certificate issues... 
I have properly set the DNS for my custom domain with 4 DNS of type A and one DNS of type CNAME redirecting my www address to  <organization>.github.io 

However, I cannot get access to the www address as of now : NET::ERR_CERT_COMMON_NAME_INVALID while the naked address is working perfectly.
Could you provide some help on this? 

Thanks!

**Comment by yoannchaudet** [comment]
If your DNS is probably configured, accessing the Pages settings section where we do a DNS check for your custom domain is enough. When we detect a change, we provision a new TLS certificate for you.

If that does not help, provide here your custom domain or reach out to support and we can look at it.
